lib/block.ml

```diff
@@ -219,7 +219,7 @@ let ends_by_semi_semi c =
   match List.rev c with
   | h :: _ ->
       let len = String.length h in
-      len > 2 && h.[len - 1] = ';' && h.[len - 2] = ';'
+      len > 2 && Astring.String.is_suffix ~affix:";;" h
```

Comment:
Why `len > 2` rather than `len >= 2` ? (the latter is implicitly checked by `is_suffix`)

Reply:
I assumed that it also checks that there is *something* in there besides just the `;;` but this might have been a typo in the original code. Maybe it really was just a guard for the string subindexing. Removing the check altogether makes the tests all pass so I've amended the code to remove the check.